default/python/AI/PlanetUtilsAI.py

```diff
@@ -180,10 +199,47 @@ def get_planet_opinion(feature: str) -> Opinion:
     return _calculate_get_planet_opinions().get(feature, default)
 
 
+@cache_for_current_turn
+def _planned_species() -> Mapping[PlanetId, str]:
+    universe = fo.getUniverse()
+    production_queue = fo.getEmpire().productionQueue
+    planned_species = {}
+    building_prefix = "BLD_COL_"
+    species_prefix = "SP_"
+    colonisation_plans = get_aistate().colonisablePlanetIDs
+    for element in production_queue:
+        if element.name.startswith(building_prefix):
+            planned_species[element.locationID] = species_prefix + element.name[len(building_prefix) :]
+    for pid in get_owned_planets_by_empire():
+        planet = universe.getPlanet(pid)
+        for building in map(universe.getBuilding, planet.buildingIDs):
+            if building.name.startswith(building_prefix):
```

Comment:
What kind of planet is it?  Outpost with colony building, which will be converted on next turn?